Userland/Libraries/LibGL/CMakeLists.txt

```diff
@@ -27,3 +27,12 @@ set(GENERATED_SOURCES
 
 serenity_lib(LibGL gl)
 target_link_libraries(LibGL PRIVATE LibGfx LibGLSL LibGPU)
+
+# Install symlinks at some common locations so ports can find LibGL without issue
+if (NOT BUILD_LAGOM)
+    add_custom_target(LibGL_symlink_include_dir ALL ${CMAKE_COMMAND} -E create_symlink LibGL/GL/ ${CMAKE_INSTALL_PREFIX}/usr/include/GL
+        DEPENDS LibGL)
+    add_custom_target(LibGL_symlink_library ALL ${CMAKE_COMMAND} -E create_symlink libgl.so.serenity ${CMAKE_INSTALL_PREFIX}/usr/lib/libGL.so
+        COMMAND ${CMAKE_COMMAND} -E create_symlink libgl.so.serenity ${CMAKE_INSTALL_PREFIX}/usr/lib/libGL.so.1
+        DEPENDS LibGL)
```

Comment:
These will be executed at build time, rather than install time, which is not great as they are using CMAKE_INSTALL_PREFIX to do so.

I'd rather add an `install(SCRIPT)` or `install(CODE)` step do to this.

Reply:
I went with `install(CODE)`; was this what you meant?

Reply:
Almost! Ref this commit I did a while back

https://github.com/ADKaster/serenity/commit/46b86ffcfcc5959ed351da252dfea4ae1208d8f9

you can do it in one statement like zo. and it's more correct to not expand CMAKE_INSTALL_DIR until actual install time, as you can do `cmake --install --prefix <some prefix that's different than build time's CMAKE_INSTALL_PREFIX>`

```cmake
    install(CODE "
      file(CREATE_LINK LibGL/GL/ \${CMAKE_INSTALL_PREFIX}/usr/include/GL SYMBOLIC)
      file(CREATE_LINK libgl.so.serenity \${CMAKE_INSTALL_PREFIX}/usr/lib/libGL.so SYMBOLIC)
      file(CREATE_LINK libgl.so.serenity \${CMAKE_INSTALL_PREFIX}/usr/lib/libGL.so.1 SYMBOLIC)
    ")
```

Reply:
Ooh that makes sense. Thanks a bunch!